CHANGELOG.md

```diff
@@ -1,217 +1,241 @@
+For an overview of this framework, check out the new [Extensibility framework][] topic on the docs site.
 
-One of the bigger changes is the migration to [Apollo Client 3.0][].
-This version of the Apollo Client provides better cache controls and better network fetching performance in general.
+In previous releases, Peregrine talons had limited Target coverage.
+This release adds an automatic API generator to Peregrine that exposes all hooks and talons as Targets.
+This means that existing and future hooks and talons in Peregrine automatically get their own Targets that developers may use for modifying or extending functionality.
 
-This release also includes various refactors and improvements on the GraphQL queries themselves to reduce the API request times from Magento.
+This release also adds the _Targetables_ feature to the extensibility framework.
+These represent source files used in your PWA Studio project, and
+they give developers the ability to change the source code during the build process.
+This means that developers no longer have to copy PWA Studio source code into their storefront projects to make minor modifications.
 
-[apollo client 3.0]: https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/
+[extensibility framework]: <https://pwastudio.io/pwa-buildpack/extensibility-framework/>
 
-### Complete cart and checkout experience
+### PWA Studio extensions
 
-This release finishes the full page cart and checkout features introduced in previous releases.
-The complete cart and checkout workflow is based on research made by members of the UX team.
+PWA Studio's extensibility framework lets developers create extensions and install them as project dependencies in their storefronts.
+This release converts some existing Venia features into PWA Studio extensions.
+It also introduces new extensions that provide useful Venia features that developers can add to their projects.
 
-Storefront developers can use this streamlined process as implemented or they can use the different components to customize their own cart and checkout workflow.
+Developers can find the source code for these extensions under the [`packages/extensions`][] directory in the PWA Studio repository.
 
-#### Shopping Bag feature
+`upward-security-headers`
+: intercepts build targets to add security headers to UPWARD
 
-In addition to the complete full page cart and checkout experience, this release introduces a new Mini-Cart/Shopping Bag feature.
+`venia-adobe-data-layer`
+: provides Adobe Client Data Layer support for your project
 
-This feature is a floating modal that appears when you click on the shopping bag icon.
-It replaces the old `MiniCart` component, which previously appeared as a drawer from the right side of the app.
+`venia-sample-backends`
+: provides demo backends and backend validation utils for your project
```

Comment:
Maybe add reminder that this extension should be removed before going live.

Reply:
Do we like shortening words in our public dev docs? Calling these utilities would be more appropriate I think, but not overly fussed. I would also aim to describe this in a bit more detail, as demo back-end may not make sense to everyone.